integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLifecycleSampleScripts.java

```diff
@@ -0,0 +1,448 @@
+  public static final String CLUSTER_LIFECYCLE = "Cluster";
+  public static final String DOMAIN = "DOMAIN";
+  public static final String STOP_SERVER_SCRIPT = "stopServer.sh";
+  public static final String START_SERVER_SCRIPT = "startServer.sh";
+  public static final String STOP_CLUSTER_SCRIPT = "stopCluster.sh";
+  public static final String START_CLUSTER_SCRIPT = "startCluster.sh";
+  public static final String STOP_DOMAIN_SCRIPT = "stopDomain.sh";
+  public static final String START_DOMAIN_SCRIPT = "startDomain.sh";
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static String domainName = "domain1";
+  private final int replicaCount = 2;
+  private final String clusterName = "cluster-1";
+  private final String adminServerName = "admin-server";
+  private final String managedServerNameBase = "managed-server";
+
+  private final String adminServerPodName = domainName + "-" + adminServerName;
+  private final String managedServerPodNamePrefix = domainName + "-" + managedServerNameBase;
+  private final Path samplePath = Paths.get(ITTESTS_DIR, "../kubernetes/samples");
+  private final Path domainLifecycleSamplePath = Paths.get(samplePath + "/scripts/domain-lifecycle");
+  private final Path tempSamplePath = Paths.get(WORK_DIR, "sample-testing");
+  private final Path sampleBase =
+          Paths.get(tempSamplePath.toString(), "scripts/create-weblogic-domain/domain-home-on-pv");
+
+  private static final String[] params = {"wlst:domain1"};
+
+  // create standard, reusable retry/backoff policy
+  private static final ConditionFactory withStandardRetryPolicy
+      = with().pollDelay(2, SECONDS)
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(10, MINUTES).await();
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Assigns unique namespaces for operator and domains and installs operator.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+
+    logger.info("Assign a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    opNamespace = namespaces.get(0);
+    logger.info("Assign a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace is null");
+    domainNamespace = namespaces.get(1);
+
+    // create pull secrets for WebLogic image when running in non Kind Kubernetes cluster
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(domainNamespace);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    //create and start WebLogic domain using domain-home-on-pv sample scripts
+    createDomain(sampleBase);
+  }
+
+  /**
+   * Test scripts for stopping and starting a managed server.
+   */
+  @ParameterizedTest
```

Comment:
Are you planning to add more domain types for these tests? If not you don't need to Parameterize the tests, since only 1 domain is created.

Reply:
Removed the ParameterizedTest annotation.